### DISCUSSION TITLE
Hardcoded issue?

### DISCUSSION THREAD

**Discussion by Keweid** [wg-easy/wg-easy#167] [General]
Is this line's port hardcoded: https://github.com/WeeJeWel/wg-easy/blob/master/src/config.js#L21 (https://github.com/WeeJeWel/wg-easy/blob/bfb9296fdb818e323663f42e7376dd271bb0f4d4/src/config.js#L21)?

**Comment by gmcinalli** [comment]
Yes it is, you can map it with Docker to every port you want.

**Comment by pheiduck** [comment]
@Keweid Yes, due Wireguard only listen on this Ports and can't on another.
